docs/index.rst

```diff
@@ -7,13 +7,46 @@ Welcome to QENSmodels's documentation!
 ======================================
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Contents:
+   :maxdepth: 3
+   :caption: Contents
 
    description
-   modules
+
+.. toctree::
+   :maxdepth: 3
+   :caption: Information on models and units
+
+   handling_units
    tools
 
+.. toctree::
+   :caption: Examples
+   :maxdepth: 1
+
+   examples/README
```

Comment:
I think when the example names are displayed in the TOC they might be a little long.  It is good that they are descriptive but I find them a little hard to read within the TOC.  Perhaps this level of detail should be captured within the Introduction (which could even be renamed Introduction and Examples Summary) and the names of the examples themselves reduced a little?

Reply:
I shortened the titles of the notebooks.